docker.Makefile

```diff
@@ -27,7 +27,10 @@ build_cross_image:
 
 # build executable using a container
 binary: build_docker_image
-	docker run --rm -e VERSION=$(VERSION) $(MOUNTS) $(DEV_DOCKER_IMAGE_NAME) make binary
+	docker run --rm \
+		-e VERSION=$(VERSION) \
+		$(if $(GITCOMMIT), -e GITCOMMIT) \
+		$(MOUNTS) $(DEV_DOCKER_IMAGE_NAME) make binary
```

Comment:
This `if` is unnecessary, it works just fine with `-e GITCOMMIT`.

Since this is repeated in a few places could we do like `$(MOUNTS)` and put this at the top with version?

``
ENVVARS := "-e VERSION=... -e GITCOMMIT"

...

docker run --rm $(ENVVARS)

```